templates/default/slurm_parallelcluster_nodes.conf.erb

```diff
@@ -0,0 +1,8 @@
+<%
+  def append_dummy
+    if node['cfncluster']['cfn_max_queue_size'].to_i > 0
+        "dummy-compute[1-" + node['cfncluster']['cfn_max_queue_size'] + "]"
+    end
+  end
+%>
+NodeName=<%= append_dummy %> CPUs=2048 State=FUTURE
```

Comment:
I see you changed from Procs to CPUs, is there any difference?
Do you think to set the right number of CPUs for the configured instance type in future?

Reply:
No difference. From what I understood Procs is the legacy parameter, CPU the newer one.

Yes, I will address the CPU configuration separately.